tests/backup/backup_helper.go

```diff
@@ -2305,3 +2305,20 @@ func RemoveLabelFromNodesIfPresent(node node.Node, expectedKey string) error {
 	}
 	return nil
 }
+
+// ValidatePodByLabel validates if the pod with specified label is in a running state
+func ValidatePodByLabel(label map[string]string, namespace string, timeout time.Duration, retryInterval time.Duration) error {
+	log.Infof("Checking if pods with label %v are running in namespace %s", label, namespace)
+	pods, err := core.Instance().GetPods(namespace, label)
+	if err != nil {
+		return err
+	}
+	for _, pod := range pods.Items {
+		err = core.Instance().ValidatePod(&pod, timeout, retryInterval)
+		if err != nil {
+			log.Errorf("Failed to validate pod [%s]", pod.GetName())
+			return err
```

Comment:
Wouldn't it be better to combine the error and return it. (`return fmt.Errorf("Faled.... %v", err)`)

Reply:
Added the return as follows:

```
return fmt.Errorf("failed to validate pod [%s] with error - %s", pod.GetName(), err.Error())
```